blob/gcsblob/gcsblob.go

```diff
@@ -110,11 +110,13 @@ func (b *bucket) NewRangeReader(ctx context.Context, key string, offset, length
 		}
 		return nil, err
 	}
+	// updated is set to zero value when non-nil error returned, no need to check or report.
+	updated, _ := r.LastModified()
```

Comment:
Hmm, so this always existed? I thought we were blocked on the upstream bug...?

Reply:
No, that issue is just fixed. I also updated go.mod file to bump the dep version.

Reply:
Oh, sorry, I missed that diff. Got it.